first_hour_on_vwb/working_with_data_collections.ipynb

```diff
@@ -43,15 +43,18 @@
    "source": [
     "### Objective\n",
     "\n",
-    "Perform common workspace resource operations including:\n",
+    "This notebook will guide you through creating a new data collection or converting an existing workspace into a data collection which you can then share with collaborators and use in your workspaces.\n",
     "\n",
-    "1. Create a new data collection from cloud data.\n",
-    "1. Share the data collection with collaborators.\n",
-    "1. Add the data collection as a resource to your workspace.\n",
+    "- [Create a new data collection](#create-new-dc)\n",
```

Comment:
Is there anything I need to do to get these links to work? When I click on them, nothing happens for me

Reply:
I used 'href' instead of 'id'. I have now fixed these and tested them.